src/VisualStudio/TestUtilities/Common/Parameter.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Language.Intellisense;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Common
+{
+    [Serializable]
+    public class Parameter : IEquatable<Parameter>
+    {
+        public string Name { get; set; }
+        public string Documentation { get; set; }
+
+        public Parameter()
+        {
+        }
+
+        public Parameter(IParameter actual)
```

Comment:
D'oh!

On Jun 23, 2016 5:11 AM, Dustin Campbell <notifications@github.com> wrote:

In src/VisualStudio/TestUtilities/Common/Parameter.cs<https://github.com/dotnet/roslyn/pull/11735#discussion_r68222974>:

> +using System;
> +using Microsoft.VisualStudio.Language.Intellisense;
> +
> +namespace Roslyn.VisualStudio.Test.Utilities.Common
> +{
> +    [Serializable]
> +    public class Parameter : IEquatable<Parameter>
> +    {
> +        public string Name { get; set; }
> +        public string Documentation { get; set; }
> +
> +        public Parameter()
> +        {
> +        }
> +
> +        public Parameter(IParameter actual)


IParameter is from the Editor API.

—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly, view it on GitHub<https://github.com/dotnet/roslyn/pull/11735/files/6e2373b9e81d4e6d585c0b9ddd1fd58bbad18aed#r68222974>, or mute the thread<https://github.com/notifications/unsubscribe/AEWmYwVtvt2HXDtj8SC8wuOlRDJmjyc1ks5qOnfpgaJpZM4ItFlT>.